hindi/Ramayana/ramayana.md

```diff
@@ -0,0 +1,3 @@
+Ramayana is one of the two major Sanskrit epics of ancient India, the other being the [Mahabharat](../Mahabharat/mahabharat.md). Along with the Mahabharat, it forms the Hindu Itihasa.
+
+To read more about Ramayana , [Ramayana](https://en.wikipedia.org/wiki/Ramayana) [रामायण](https://hi.wikipedia.org/wiki/%E0%A4%B0%E0%A4%BE%E0%A4%AE%E0%A4%BE%E0%A4%AF%E0%A4%A3)
```

Comment:
This line contains more than one link.  Please put links on separate lines with a blank line in between so that options will be visually separated.

Reply:
@casey-collab